Pspcl/Pspcl.Models/Pspcl.Models.csproj

```diff
@@ -6,4 +6,8 @@
     <Nullable>enable</Nullable>
   </PropertyGroup>
 
+  <ItemGroup>
```

Comment:
Why we have installed rest sharp in models?

Reply:
I uninstalled it from models